### PR TITLE
feat: minor adjustments in archetype design

### PR DESCRIPTION
**Pull request by lauramargar**
# Pull request template
 
Restyle with minor adjustments the archetype following the designs: https://www.figma.com/file/mYfJZHn668mGEGf61Sji6m/X-Archetype?node-id=72%3A468&mode=dev

## Motivation and context

- [ ] Dependencies. If any, specify:
- [X] Open issue. If applicable, link: [EMP-771](https://searchbroker.atlassian.net/jira/software/c/projects/EMP/boards/294?modal=detail&selectedIssue=EMP-771&quickFilter=1099)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [X] `Main`
- [ ] Other. Specify:

## How has this been tested?

Follow the task description.

## Checklist:

- [ ] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

[EMP-771]: https://searchbroker.atlassian.net/browse/EMP-771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/my-history/custom-my-history.vue [hunk 1]
```diff
@@ -38,15 +38,7 @@
...
-      <div v-else class="x-flex x-flex-1 x-flex-col x-items-center">
-        <NoHistoryIcon
-          class="x-flex-no-shrink"
-          :class="{ 'x-grayscale': !$x.isHistoryQueriesEnabled }"
-        />
-        <p class="x-title3 x-py-16 x-text-neutral-75">
-          {{ $t('myHistory.noHistory') }}
-        </p>
-      </div>
+      <div v-else />
```
Comment (herrardo) [inline src/components/my-history/custom-my-history.vue:41]: Why are we assuming that we have to delete the My History graphic based on the designs?

https://www.figma.com/file/mYfJZHn668mGEGf61Sji6m/X-Archetype?node-id=3027%3A22233&mode=dev

There are two versions in there..

File: src/components/search/results/custom-next-query-preview.vue [hunk 2]
```diff
@@ -5,14 +5,21 @@
...
       <template #query>
         <span class="x-title3">{{ $x.query.nextQueries }}</span>
       </template>
     </i18n>
     <CustomSlidingPanel>
       <template #header>
-        <NextQuery :suggestion="nextQuery" class="x-button-tight x-button max-desktop:x-px-16">
+        <NextQuery
+          :suggestion="nextQuery"
+          class="x-button-tight x-button x-text-lead-50 max-desktop:x-px-16"
```
Comment (CachedaCodes) [inline src/components/search/results/custom-next-query-preview.vue:21]:
```suggestion
          class="x-button-tight x-button x-button-lead max-desktop:x-px-16"
```

File: src/components/search/results/partial-results.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <template #header>
           <PartialQueryButton
             :query="partialResult.query"
-            class="x-button-tight x-button max-desktop:x-px-16"
+            class="x-button-tight x-button x-text-lead-50 max-desktop:x-px-16"
```
Comment (CachedaCodes) [inline src/components/search/results/partial-results.vue:8]:
```suggestion
            class="x-button-tight x-button x-button-lead max-desktop:x-px-16"
```

File: src/components/search/no-results-message.vue [hunk 4]
```diff
@@ -1,22 +1,27 @@
 <template>
   <i18n
     v-if="$x.noResults"
-    class="x-no-results-message x-text1 x-flex x-flex-col x-items-center x-gap-8 x-break-words x-bg-neutral-10 x-p-24 x-text-center desktop:x-text1-lg"
+    class="x-no-results-message x-text1 x-mt-24 x-flex x-items-center x-break-words x-bg-neutral-10 x-py-8 x-text-center x-text-neutral-90 desktop:x-flex-wrap desktop:x-justify-center desktop:x-gap-8 desktop:x-text1-lg desktop:x-py-24"
```
Comment (CachedaCodes) [inline src/components/search/no-results-message.vue:4]: Why are we changing so many text colors from `x-text1` to `x-text-neutral-90`?

File: src/components/search-box.vue [hunk 5]
```diff
@@ -16,9 +16,9 @@
 
     <SearchButton
       v-if="isDesktopOrGreater || !$x.query.searchBox"
-      class="x-input-group-button-primary"
+      class="x-input-group-button-primary x-bg-neutral-90"
```
Comment (CachedaCodes) [inline src/components/search-box.vue:19]: This doesn't work design wise, we don't have a button with a background color of `neutral-90`. We lose the hovering interactions with this.

**Review by CachedaCodes — CHANGES_REQUESTED** [review]

File: src/components/search/no-results-message.vue (hunk 4 above)
Comment (lauramargar) [reply src/components/search/no-results-message.vue:4]: Many texts need the color `x-text-neutral-90`, I mean this color `#283034`, but by default, all these texts have associated this color `#0000`. We could separate these text changes in another task and investigate how to fix them.

File: public/snippet-script.js [hunk 6]
```diff
@@ -63,7 +62,6 @@ window.initX = {
   env,
   scope,
   lang,
-  device,
```
Comment (CachedaCodes) [inline public/snippet-script.js:66]: Restore this

**Review by CachedaCodes — CHANGES_REQUESTED** [review]

**Review by CachedaCodes — APPROVED** [review]

**Pull request merged**